### PR TITLE
[docs] copy edit of Job Submission Getting Started and FAQ

### PR DESCRIPTION
**Pull request by angelinalg**
Quick follow to https://github.com/ray-project/ray/pull/41319

## Checks

- [x] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: doc/source/cluster/running-applications/job-submission/quickstart.rst [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
+This guide walks through the Ray Jobs CLI commands available for submitting and interacting with a Ray Job.
 
-To use the Jobs API programmatically via a Python SDK instead of a CLI, see :ref:`ray-job-sdk`.
+To use the Jobs API programmatically with a Python SDK instead of a CLI, see :ref:`ray-job-sdk`.
 
 Setup
 -----
 
-Ray Jobs is available in versions 1.9+ and requires a full installation of Ray. You can do this by running:
+Ray Jobs is available in versions 1.9+ and requires a full installation of Ray. You can do install Ray by running:
```
Comment (architkulkarni) [inline doc/source/cluster/running-applications/job-submission/quickstart.rst:15]:
```suggestion
Ray Jobs is available in versions 1.9+ and requires a full installation of Ray. You can install Ray by running:
```

File: doc/source/cluster/running-applications/job-submission/quickstart.rst [hunk 2]
```diff
@@ -203,40 +203,40 @@ Finally, if we want to cancel the job, we can use ``ray job stop``:
...
 
-Using a Remote Cluster
+Using a remote cluster
 ----------------------
 
-The example above was for a local Ray cluster.  When connecting to a `remote` cluster, you need to be able to access the dashboard port of the cluster over HTTP.
+The preceding example is for a local Ray cluster. When connecting to a `remote` cluster, you need to access the dashboard port of the cluster over HTTP.
 
-One way to do this is to port forward ``127.0.0.1:8265`` on your local machine to ``127.0.0.1:8265`` on the head node. If you started your remote cluster with the :ref:`Ray Cluster Launcher <cluster-index>`, then the port forwarding can be set up automatically using the ``ray dashboard`` command (see :ref:`monitor-cluster` for details).
+One way to do access the port is to port forward ``127.0.0.1:8265`` on your local machine to ``127.0.0.1:8265`` on the head node. If you started your remote cluster with the :ref:`Ray Cluster Launcher <cluster-index>`, then you can set up automatic port forwarding using the ``ray dashboard`` command. See :ref:`monitor-cluster` for details.
```
Comment (architkulkarni) [inline doc/source/cluster/running-applications/job-submission/quickstart.rst:211]:
```suggestion
One way to access the port is to port forward ``127.0.0.1:8265`` on your local machine to ``127.0.0.1:8265`` on the head node. If you started your remote cluster with the :ref:`Ray Cluster Launcher <cluster-index>`, then you can set up automatic port forwarding using the ``ray dashboard`` command. See :ref:`monitor-cluster` for details.
```

File: doc/source/cluster/running-applications/job-submission/quickstart.rst [hunk 3]
```diff
@@ -251,7 +251,7 @@ The Ray Jobs API provides an option to specify the runtime environment when subm
     ray.init()
     print("requests version:", ray.get(get_requests_version.remote()))
 
-First, let's submit this job using the default environment. This is the environment in which the Ray Cluster was started.
+Ssubmit this job using the default environment. This environment is the environment you started the Ray Cluster in.
```
Comment (architkulkarni) [inline doc/source/cluster/running-applications/job-submission/quickstart.rst:254]:
```suggestion
Submit this job using the default environment. This environment is the environment you started the Ray Cluster in.
```

**Review by architkulkarni — COMMENTED** [review]
Thanks! Just a few typos

**Comment by angelinalg** [comment]
Thanks for catching those, @architkulkarni! Eagle eyes!

**Comment by architkulkarni** [comment]
@angelinalg doc build fails with 

> /home/docs/checkouts/readthedocs.org/user_builds/anyscale-ray/checkouts/41342/doc/source/cluster/running-applications/job-submission/quickstart.rst:312: WARNING: Inline interpreted text or phrase reference start-string without end-string.

Any ideas?

File: doc/source/cluster/running-applications/job-submission/quickstart.rst [hunk 4]
```diff
@@ -308,6 +308,6 @@ Now let's try it with a runtime environment that pins the version of the ``reque
     If both the Driver and Job specify a runtime environment, Ray tries to merge them and raises an exception if they conflict.
     See :ref:`runtime environments <runtime-environments-job-conflict>` for more details.
 
-- The full API reference for the Ray Jobs CLI can be found :ref:`here <ray-job-submission-cli-ref>`. 
-- The full API reference for the Ray Jobs SDK can be found :ref:`here <ray-job-submission-sdk-ref>`.
-- For more information, check out the guides for :ref:`programmatic job submission <ray-job-sdk>` and :ref:`job submission using REST <ray-job-rest-api>`.
+- See :ref:`Ray Jobs CLI <ray-job-submission-cli-ref>` for a full API reference of the CLI. 
+- See :ref:`Ray Jobs SDK <ray-job-submission-sdk-ref>`for a full API reference of the SDK.
```
Comment (architkulkarni) [inline doc/source/cluster/running-applications/job-submission/quickstart.rst:312]:
```suggestion
- See :ref:`Ray Jobs SDK <ray-job-submission-sdk-ref>` for a full API reference of the SDK.
```

Comment (architkulkarni) [reply doc/source/cluster/running-applications/job-submission/quickstart.rst:312]: Not sure if this is related ot the doc build failure, not sure what could be causing it

**Comment by angelinalg** [comment]
Thanks, @archit, for investigating the failure. I've accepted your suggestion to kick off a new build.

**Comment by angelinalg** [comment]
@architkulkarni, I think you fixed it!

**Review by architkulkarni — APPROVED** [review]

**Pull request merged**